### PR TITLE
Feature/kal button

### PR DESCRIPTION
**Pull request by KALjpo**

### CODE REVIEW

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-button/kal-button.component.sass [hunk 1]
```diff
@@ -1,5 +1,51 @@
...
   kal-button
-    input
-      border: none
+    $self: &
+    display: inline-block
+    &:focus
+      outline: none
+
+    +kal-themed-button(get('form.elements.button.default', $kal-parameters))
+    &.#{$self}--primary
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-button/kal-button.component.sass]: same as `&--primary`

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-button/kal-button.component.sass [hunk 2]
```diff
@@ -1,5 +1,51 @@
...
+      +kal-themed-button(get('form.elements.button.primary', $kal-parameters))
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-button/kal-button.component.sass]: should use @each : 
```sass
@each $key, $params in get('form.elements.button.variantes', $kal-parameters)
  &--#{$key}
    +kal-themed-button($params)
```

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-button/kal-button.component.sass [hunk 3]
```diff
@@ -1,5 +1,51 @@
...
+    &.#{$self}--secondary
+      +kal-themed-button(get('form.elements.button.secondary', $kal-parameters))
+    &.#{$self}--tertiary
+      +kal-themed-button(get('form.elements.button.tertiary', $kal-parameters))
+    &.#{$self}--reverse
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-button/kal-button.component.sass]: add reverse in kal-themed-button mixin

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-menu/kal-menu.component.html [hunk 4]
```diff
@@ -1,8 +1,8 @@
 <img
   *ngIf="logo"
   [src]="logo"
-  width="100"
-  height="100"/>
+  width="150px"
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-menu/kal-menu.component.html]: image should be 100% width and liimted in main project using library

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-menu/kal-menu.component.html [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
+  height="150px"/>
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-menu/kal-menu.component.html]: do not provide height, for auto scalling

File: projects/kalidea/kaligraphi/src/styles/_parameters.scss [hunk 6]
```diff
@@ -33,13 +35,174 @@ $base-border-width: 1px !default;
...
+    'hover': (
+      'background-color': get("grey.wf100", $kal-palettes),
+      'border': 1px solid get("grey.wf100", $kal-palettes)
+    ),
+    'active': (
+      'background-color': get("grey.wf300", $kal-palettes),
+      'border': 1px solid get("grey.wf300", $kal-palettes)
+    )
+  ),
+  'primary': (
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/styles/_parameters.scss]: put all themes in sub key `variantes` or `themes`

File: projects/kalidea/kaligraphi/src/styles/_parameters.scss [hunk 7]
```diff
@@ -33,13 +35,174 @@ $base-border-width: 1px !default;
...
+    'hover': (
+      'background-color': get("grey.wf100", $kal-palettes),
+      'border': 1px solid get("grey.wf300", $kal-palettes)
+    ),
+    'active': (
+      'background-color': get("grey.wf200", $kal-palettes),
+      'border': 1px solid get("grey.wf300", $kal-palettes)
+    )
+  ),
+  'reverse': (
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/styles/_parameters.scss]: put each reverse in related variante

File: src/app/molecules/button/button.component.html [hunk 8]
```diff
@@ -0,0 +1,134 @@
+<h1>kal-button</h1>
```
Comment (ydomenjoud) [inline src/app/molecules/button/button.component.html:1]: provide tool to edit theme of this button on the fly

**Review by ydomenjoud — CHANGES_REQUESTED** [review]

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-button/kal-button.component.sass (hunk 1 above)
Comment (KALjpo) [reply projects/kalidea/kaligraphi/src/lib/molecules/kal-button/kal-button.component.sass]: non, car du coup
**kal-button--primary button** est moins fort que  **kal-button button**
il faut  **kal-button.kal-button--primary button**

**Review by ydomenjoud — APPROVED** [review]

**Pull request merged**